src/main/java/cn/nukkit/Player.java

```diff
@@ -1834,7 +1834,7 @@ public void checkNetwork() {
     }
 
     public boolean canInteract(Vector3 pos, double maxDistance) {
-        return this.canInteract(pos, maxDistance, 0.6);
+        return this.canInteract(pos, maxDistance, 8.0);
```

Comment:
Isn't it something between 3 and 5 in vanilla?

Reply:
In the vanilla code it's 6.